sdk/mx.sdk/mx_sdk_vm_impl.py

```diff
@@ -781,6 +781,7 @@ def _get_metadata(suites, parent_release_file=None):
         _metadata_dict.setdefault('JAVA_VERSION', _src_jdk.version)
         _metadata_dict.setdefault('OS_NAME', get_graalvm_os())
         _metadata_dict.setdefault('OS_ARCH', mx.get_arch())
+        _metadata_dict.setdefault('OS_VARIANT', mx.get_os_variant())
```

Comment:
I am wondering if we should skip this when `mx.get_os_variant()` is an empty string